kolibri/plugins/coach/assets/src/views/plan/assignments/AssignmentDetailsModal.vue

```diff
@@ -217,14 +218,14 @@
       assignmentStrings() {
         if (this.assignmentIsQuiz) {
           return {
-            activeStatus: this.coachCommon$tr('quizActiveLabel'),
-            inactiveStatus: this.coachCommon$tr('quizInactiveLabel'),
+            activeStatus: this.coachString('activeLabel'),
+            inactiveStatus: this.coachString('inactiveLabel'),
             statusExplanation: this.$tr('activeQuizzesExplanation'),
           };
         }
         return {
-          activeStatus: this.coachCommon$tr('lessonActiveLabel'),
```

Comment:
Revert this to preserve adjective-noun agreement I mentioned above. Same with quizzes.